javascript/javascript1/week3/warmUp.js

```diff
@@ -0,0 +1,79 @@
+// Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+
+  // Write some code here
+  const indexOfNameToRemove = names.indexOf(nameToRemove);
+  const numberOfNameToRemove = 1;
+  names.splice(indexOfNameToRemove, numberOfNameToRemove)
+  // Code done
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+//   When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function calculateTravelTime(informationInput){
+    calculateTime = informationInput.destinationDistance / informationInput.speed
```

Comment:
The use of `parseInt` and `Number` should not be necessary here. Can you see why? 🔍 

It is also important to remember to use `let` or `const` for your variables! Which would you choose here? 😄

Reply:
So the here is it better to use `Math.floor(calculateTime)` ?
Maybe `let` in the function?

Reply:
Everything should already be and stay a number so `parseInt` and `Number` at best should do nothing or they might break stuff. I think putting a number into `parseInt` might convert it to a string and back to a number, but maybe it is smart! 😄

For `const`/`let`, I'd suggest always starting with `const` and then switch to `let` if you find yourself needing to modify it. (It is good practice to see if you can keep something `const` as it is easier to maintain code if things cannot change in weird places.) 👍